nncf/common/graph/transformations/commands.py

```diff
@@ -35,6 +35,7 @@ class TransformationPriority(IntEnum):
     FP32_TENSOR_STATISTICS_OBSERVATION = 1
     PRUNING_PRIORITY = 2
     SPARSIFICATION_PRIORITY = 3
+    OP_INSERTION_PRIORITY = 4
```

Comment:
Comment from the original PR, @KodiaqQ:
Why do we need a new priority? For which operations?

Reply:
@KodiaqQ, you are right! Let me remove it

Reply:
Done